pkg/meta/repodb/dynamodb-wrapper/dynamo_wrapper.go

```diff
@@ -481,79 +481,25 @@ func (dwr DBWrapper) DeleteReferrer(repo string, referredDigest godigest.Digest,
 	return dwr.setRepoMeta(repo, repoMeta)
 }
 
-func (dwr DBWrapper) GetFilteredReferrersInfo(repo string, referredDigest godigest.Digest,
+func (dwr DBWrapper) GetReferrersInfo(repo string, referredDigest godigest.Digest,
 	artifactTypes []string,
 ) ([]repodb.ReferrerInfo, error) {
-	referrersDescriptors, err := dwr.GetReferrers(repo, referredDigest)
+	referrersInfo, err := dwr.GetReferrers(repo, referredDigest)
 	if err != nil {
 		return nil, err
 	}
 
-	referrersInfo := []repodb.ReferrerInfo{}
```

Comment:
Where has this logic moved?

Reply:
Ok, I did not notice OnUpdateManifest was calling SetMetadataFromInput which was calling GetReferredSubject.